### DISCUSSION TITLE
Adminutils installation failed | Ed Easy Installer

### DISCUSSION THREAD

**Discussion by Marc70008** [Sunbird-Ed/Community#538] [Installation] [answered]
Hi, I am following the Ed Easy Installer script to provision Ed platform
While running the Ed easy installer bash script, Adminutils(Lern BB)  installation failed due to `'unable to build kubernetes objects from release manifest: error validating "": error validating data: [unknown object type "nil" in ConfigMap.data.LEARNER_API_AUTH_KEY, unknown object type "nil" in ConfigMap.data.REFRESH_TOKEN_SECRET_KEY]"`

What should be the values for the above(LEARNER_API_AUTH_KEY and REFRESH_TOKEN_SECRET_KEY)

**Comment by Krishnaj20** [comment]
@gandham-santhosh - for your attention please.

**Reply by tsprasath** [reply, marked as answer]
As part of onboard consumer jobs you will a JWT token generated for `adminutil_learner_api_key` refer to the `LEARNER_API_AUTH_KEY`
`REFRESH_TOKEN_SECRET_KEY` Go to postgresql pod --> select keycloak db--> Run this psql command 
`SELECT value FROM component_config CC INNER JOIN component C ON(CC.component_id = C.id) WHERE C.realm_id = 'sunbird' and provider_id = 'hmac-generated' AND CC.name = 'secret';`

**Reply by Marc70008** [reply]
Thanks Prasath, that worked.
One clarification required:
In the postscript function (inside install-ed.sh), the string used to fetch the JWT token for admin-utils is "JWT token for api-admin is" but it should be "JWT token for adminutil_learner_api_key is" right??
